src/controllers/authController.js

```diff
@@ -0,0 +1,31 @@
+import models from "../database/models/index.js";
+const {Users} = models;
+import helper from "../utils/jwtHelper.js";
+const { comparePassword } = require("../utils");
+class User{
+  async login(req, res, next){
```

Comment:
```suggestion
 static async login(req, res, next){
```
so that the method can be accessible in other files